client/src/utils/validations.js

```diff
@@ -6,6 +6,9 @@ export const tooLongField = (v) =>
 export const tooShortField = (v) =>
   v === '' || !!(v && v.length >= 2) || 'Field is too short';
 
+export const passwordField = (v) =>
```

Comment:
There is already a validation rule `shortPassword`

Reply:
just remove `shortPassword` cos it's not used anywhere

Reply:
Ah must have overlooked it

Reply:
np